### DISCUSSION TITLE
Pagination options in UI.table

### DISCUSSION THREAD

**Discussion by mrhunsaker** [zauberzeug/nicegui#2099] [Q&A] [answered]
Is there a way to place the arrows/icons for pagination of a table on the bottom or top left rather than the bottom right?

**Comment by falkoschindler** [comment, marked as answer]
Unfortunately Quasar's [QTable](https://quasar.dev/vue-components/table) doesn't seem to offer any customization options for this matter. So we have to get creative and find some CSS definition to fine-tune the pagination layout:

```py
ui.add_head_html('''
    <style>
    .q-table__bottom {
        justify-content: flex-start !important;
    }
    .q-table__bottom .q-table__separator {
        display: none;
    }
    </style>
''')

columns = [{'name': 'name', 'label': 'Name', 'field': 'name'}]
rows = [{'name': 'Alice'}]
ui.table(columns, rows, pagination=3).classes('w-96')
```
